definitions/npm/react-jss_v6.1.x/flow_v0.30.x-/react-jss_v6.1.x.js

```diff
@@ -8,6 +8,8 @@ declare module 'react-jss' {
 
   declare type Null = null | void;
 
+  declare type Dictionary = { [key: string]: {} }
```

Comment:
I see, I think we didn't publish lib types to npm. Mb it should be inside of flow-typed? Or what is the right approach?